platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -1,9 +1,10 @@
-import React, { Suspense, useState, useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import moment from 'moment';
 import qs from 'query-string';
+import { isEqual } from 'lodash';
```

Comment:
`loadash` is a devDependency, not a `dependency`

Reply:
Switched to lodash.isequal which is in dependencies